src/app.zig

```diff
@@ -810,6 +810,21 @@ pub const App = struct {
         }
     }
 
+    pub fn selectChannelName(self: *App, cl: *irc.Client, name: []const u8) void {
```

Comment:
Just a thought - would you be interested in making another PR to expose this to the lua api? No worries if you don't want to - just seems like it would be a good one to have there, then you could have a keybind to jump to a specific channel.